scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -287,8 +287,8 @@ task PostprocessGermlineCNVCalls {
         calls_args=""
         for index in ${dollar}{!gcnv_calls_tar_array[@]}; do
             gcnv_calls_tar=${dollar}{gcnv_calls_tar_array[$index]}
-            mkdir -p CALLS_$index/SAMPLE_${sample_index}
-            tar xzf $gcnv_calls_tar -C CALLS_$index/SAMPLE_${sample_index}
+            mkdir CALLS_$index
+            tar xzf $gcnv_calls_tar -C CALLS_$index
             cp ${dollar}{calling_configs_array[$index]} CALLS_$index/
```

Comment:
You don't need to pass as input or untar any of these anymore:
 ```
    Array[File] calling_configs
    Array[File] denoising_configs
    Array[File] gcnvkernel_version
    Array[File] sharded_interval_lists
```
Look at the diff for `cnv_common_tasks.wdl` https://github.com/broadinstitute/gatk/commit/f0dddbc686a87cf969f4be5b0594edda081690c1#diff-f3a83ecdd107a7c42aee06dc41842fd2

Reply:
Thanks for catching this.  I removed them from the PostprocessGermlineCNVCalls task but left them as outputs of the workflows.  Might be useful for debugging, but we can always change this later if you like.

Reply:
I also added the counts as outputs of the scattered case workflow.  Perhaps at some point we can flatten the outputs there.  This is trivial for all outputs except the ploidy calls, which need to be uncompressed, concatenated, and compressed again.